### PR TITLE
update exhibits cap prod deployment to automatically deploy to both -…

### PR DESCRIPTION
**Pull request by peetucket**
…a and -b servers automatically

### CODE REVIEW

File: config/deploy/production.rb [hunk 1]
```diff
@@ -1,9 +1,15 @@
-set :deploy_host, ENV['CAPISTRANO_PRODUCTION_DEPLOY_HOST'] || ask(:deploy_host, '')
+set :deploy_host, ENV['CAPISTRANO_PRODUCTION_DEPLOY_HOST'] || ask('deploy_host (leave off stanford.edu)', '')
```
Comment (ndushay) [inline config/deploy/production.rb]: perhaps the prompt should be clear that this is the prod box prefix, not the full prod box name?

File: config/deploy/production.rb [hunk 2]
```diff
@@ -1,9 +1,15 @@
...
 set :user, ENV['CAPISTRANO_PRODUCTION_USER'] || ask(:user, '')
+set :bundle_without, %w(deployment development test).join(' ')
```
Comment (ndushay) [inline config/deploy/production.rb]: i think this was happening automagically?  Not sure why - puppet config for bundler?

File: config/deploy/production.rb [hunk 3]
```diff
@@ -1,9 +1,15 @@
...
 
-server "#{fetch(:deploy_host)}", user: fetch(:user), roles: %w(web db app)
+user_server_extensions = %w(a b)
+user_server_extensions.each do |extension|
```
Comment (ndushay) [inline config/deploy/production.rb]: %(a b).each do  |suffix|  ... ?

**Comment by ndushay** [comment]
My comments are quibbly, except maybe the one about being clear we are entering the machine name prefix, not the full host name?

File: config/deploy/production.rb (hunk 2 above)
Comment (peetucket) [inline config/deploy/production.rb]: if so, that would be cool, but i think this just determines how bundler is called by capistrano when deploying - we use it in other projects i deploy with (and SW does too)

when he has a chance @cbeer can let us know

File: config/deploy/production.rb (hunk 1 above)
Comment (peetucket) [inline config/deploy/production.rb]: good idea, i'll adjust

**Comment by ndushay** [comment]
:+1:    (rebase and merge yourself)

**Comment by cbeer** [comment]
Fixed by #186

**Pull request closed without merging**